generate_models.sh

```diff
@@ -18,15 +18,20 @@ fields as (
                 staging_columns=get_$5_columns()
             )
         }}
-        
     from base
 ),
 
 final as (
     
-    select 
-    -- rename here
-    from fields
+    select " >> $1/models/$2__$5.sql
+    
+
+dbt run-operation fivetran_utils.get_column_names_only --args '{"table_name": "'$5'", "schema_name": "'$4'", "database_name":"'$3'"}' | tail -n +2 >> $1/models/$2__$5.sql
+    
+    
+echo "    from fields
 )
 
-select * from final" >> $1/models/$2__$5.sql
+select *
+from final" >> $1/models/$2__$5.sql
```

Comment:
Is this in line with our current sql style standards?

Reply:
@fivetran-joemarkiewicz so i'm basing it off the sql guide, using this as reference: https://github.com/fivetran/dbt_style_guide/blob/main/sql_guidelines.md#ctes